.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Android-ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: docker://fabernovel/android:api-29-v1.1.0
+    steps:
+      - uses: actions/checkout@v4
+      - name: set up JDK 11
```

Comment:
Why do you need to install jdk11? Looks like fabernovel/android has Java?

Reply:
removed